src/community/oseo/oseo-stac/src/main/resources/org/geoserver/ogcapi/v1/stac/items.json

```diff
@@ -18,6 +18,8 @@
         "$${maxy(footprint)}"
       ],
       "properties": {
+        "test:min": "$${eoSummaries('min',eop:identifier,'eoTrack')}",
```

Comment:
I would have expected to see the same level of abstraction as the other properties, e.g., if we use "sar:frequency_band" then the property name should also be "eop:track", no?

Reply:
Also, the aggregates should be used in collections.json, they are aggregating product values so that the collection reports them, but for a single product, there is nothing to aggregate.

Please come up with an integration test for each aggregate type, too.

Reply:
fixed